manifests/v/Veloren/Airshipper/0.6.0/Veloren.Airshipper.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Veloren.Airshipper
+PackageVersion: 0.6.0
+PackageLocale: en-US
+Publisher: Veloren

```

Comment:
```suggestion
Publisher: Airshipper contributors
```

Reply:
This must match Control Panel.

Reply:
Ok, we can change it to that, but I don't think we should because the word "veloren" needs to be there to help people find the package, almost nobody is going to search for "Airshipper", pretty much everyone is going to search for "Veloren" and since the game is being developed here: https://github.com/veloren/airshipper/ which is under this group: https://github.com/veloren
then I think we should keep it as is.
If you still think it should say "Airshipper contributors" then I'll change it but this is the argument :) What do you think now?

Reply:
You can put `veloren` in `Tags:`.

WinGet uses Control Panel entries for mapping in PackageName, Publisher, and ProductCode.

Reply:
I pushed an updated one with the changes you pointed out and added some tags (thanks for pointing those out, I didn't know about them).
I used the wingetcreate tool to create the PR.
What is the control panel?
Thanks 🙏